vsintegration/src/unittests/Tests.ProjectSystem.MSBuildUtilities.fs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace UnitTests.Tests.ProjectSystem
+
+open System
+open NUnit.Framework
+open UnitTests.TestLib.Utils
+open Microsoft.VisualStudio.FSharp.ProjectSystem
+
+
+[<TestFixture>]
+type MSBuildUtilitiesTest() = 
+    
+    static let pathSeparatorsPermutations = 
+        let pathSeparators = [""; @"\"; @"\\"] //TODO add '/'
+        [ for x in pathSeparators do 
+            for y in pathSeparators do
+                yield x, y ]
+
+    static let allPathsSeparator l =
+        seq { 
+            for x, y in pathSeparatorsPermutations do
+                for a, b, result in l |> Seq.ofList do
+                    yield a + x, b + y, result
+        }
+
+
+    static member IsSameDirTestCases =
+        [ @"c:\path\to", @"c:\path\to", true
+          @"c:\path\to", @"c:\path\to2", false
+          @"c:\path", @"c:\path", true ]
+        |> allPathsSeparator
+        |> Seq.map (fun (a,b,result) -> TestCaseAttribute(a, b, result))
+
+    [<Test; TestCaseSource("IsSameDirTestCases")>]
```

Comment:
Internally, source is in big TFS repo along with other Visual Studio stuff, and has various fixed sets of dependencies. Projects used to build shipping VF# tools are not the same as in this repo (though close), tests are run through different systems.  We are looking at unifying/simplifying, but for now NUnit stuff needs to be compatible with 2.2.2.